pkg/test/framework/components/istio/operator.go

```diff
@@ -217,14 +214,38 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 			return nil, err
 		}
 	}
+	// If the control plane cluster is not the config cluster, we are under central Istiod which uses remote KUBECONFIG
```

Comment:
I've discussed this with @linsun a bit... "central istiod" conflates the concepts of externalizing the config cluster with the new istiod-less remote cluster.

I think it would be better to decouple these concepts entirely throughout this PR (and others). The `configTopology` should be completely separate from the "type" of remote cluster used. It would be entirely feasible, for example, to have a regular primary cluster get its configuration elsewhere.

I propose that we have 2 flags to manage all of this:

- `configTopology`: configures the mapping for each cluster to its config cluster
- `useLegacyRemote`: If false (default), all remote clusters will be the new istiod-less remotes. If true, the "legacy" remote with istiod will be used.

@stevenctl @linsun @tbarrella @howardjohn WDYT?

Reply:
I kept the configTopology change in this PR for future use. This test will not rely on UseLegacyRemote instead.